docs/src/index.js

```diff
@@ -163,8 +163,11 @@ $(document).ready(() => {
           } else if (element.textContent.match(/^[0-9a-z_.:@=^]{1,30}[>|#]\s/gm)) {
             regExpCopy = /^[0-9a-z_.:@=^]{1,30}[>|#]\s/gm;
           }
-        // Don't add a copy button to blocks labeled "output"
-        } else if (['output'].includes(languageDescriptor)) {
```

Comment:
Do you think this can be rewritten as 
```
else if (['output', 'nocopy'].includes(languageDescriptor)) 
```

Reply:
The ordering is significant, so we needed to flip the order in order to accommodate the `output|nocopy <languageName>` construction. If you'd like to combine lines 168 and 170, though, go for it...

Reply:
Understood! This is good then.